src/ip_interpolator_mod.f90

```diff
@@ -1,3 +1,5 @@
+!> @file
+!! @brief 
```

Comment:
That's too brief! Put in a one-sentence description of the file. Also author and date tags (if known).